File: app/src/main/java/com/example/seki/android_custom_linklifytext_sample/ListData.java
package com.example.seki.android_custom_linklifytext_sample;

import android.databinding.ObservableField;
import android.text.Spannable;

public class ListData {
    public final ObservableField<Spannable> text = new ObservableField<>();

    public ListData(Spannable text) {
        this.text.set(text);
    }
}

File: app/src/main/java/com/example/seki/androidCustomLinklifytextSample/ListData.kt
package com.example.seki.androidCustomLinklifytextSample;

import android.databinding.ObservableField;
import android.text.Spannable;

class ListData(text: Spannable) {
    val text: ObservableField<Spannable> = ObservableField()

    init {
        this.text.set(text)
    }
}
